README.md

```diff
@@ -40,6 +40,11 @@ The generator creates and runs a migration to add the necessary table to your
 database. It also mounts Maintenance Tasks in your `config/routes.rb`. By
 default the web UI can be accessed in the new `/maintenance_tasks` path.
 
+In case you use an exception reporting service (e.g. Bugsnag) you might want to
+define an error handler. See
+[Customizing the error handler](#customizing-the-error-handler) for more
+information.
```

Comment:
```suggestion
In case you use an exception reporting service (e.g. Bugsnag) you might want to
define an error handler. See [Customizing the error
handler](#customizing-the-error-handler) for more information.
```

Reply:
I'd rather not break the link in the end.